KubeArmor/core/kubeArmor.go

```diff
@@ -461,15 +461,15 @@ func KubeArmor() {
 			}
 
 			// monitor containers
-			if strings.Contains(cfg.GlobalCfg.CRISocket, "docker") {
+			if strings.Contains(dm.Node.ContainerRuntimeVersion, "docker") {
```

Comment:
if CRI socket is not set, KubeArmor automatically detects CRI based on dm.Node.ContainerRuntimeVersion. do you think we still need to use dm.Node.ContainerRuntimeVersion even with the user-defined CRI socket?